data_dir/mv_synchronous_updates.yaml

```diff
@@ -0,0 +1,87 @@
+# LWT test: create and update data using LWT.
+### DML ###
+
+# Keyspace Name
+keyspace: mv_synchronous_ks
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE IF NOT EXISTS mv_synchronous_ks WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: mv_synchronous_cf
```

Comment:
I think it's a bit confusing to call the base table "mv_syncronus_cf".
It would be better to call it "blog_posts" or something descriptive like that.

Reply:
It means that the goal of this table is to test synchronous materialized views

Reply:
I know what it means, but it's confusing.
You see a table that is called mv_syncronous you think it's the MV.

Reply:
changed